widget/embedded/src/pages/SettingsPage.tsx

```diff
@@ -178,18 +189,52 @@ export function SettingsPage() {
         </Tooltip>
       </>
     ),
+    start: <DoneIcon color="gray" size={12} />,
     end: <Switch checked={infiniteApprove} />,
     onClick: toggleInfiniteApprove,
   };
 
+  const getThemeIcon = (theme: ThemeMode) => {
+    switch (theme) {
+      case 'auto':
+        return <AutoThemeIcon color="gray" size={12} />;
+      case 'dark':
+        return <DarkModeIcon color="gray" size={12} />;
+      default:
+        return <LightModeIcon color="gray" size={12} />;
+    }
+  };
+
+  const themeItem = {
+    id: 'theme-item',
+    type: <ListItem title="_" />,
+    hasDivider: false,
+    onClick: () => console.log(),
```

Comment:
Why do we need this? You could make it optional props if it's not used.

Reply:
fixed